client/go/outline/config/types.go

```diff
@@ -28,6 +28,7 @@ type ConnType int
 const (
 	ConnTypeDirect ConnType = iota
 	ConnTypeTunneled
+	ConnTypeBlocked
```

Comment:
We need to review every place where we set the ConnType to see if we are handling ConnTypeBlocked correctly.

For example, if the Shadowsocks subdialer is blocked, so is itself.